playbooks/roles/bats/defaults/main.yaml

```diff
@@ -4,6 +4,6 @@ bats_forklift_dir: "/root/forklift"
 bats_output_dir: "/root/bats_results"
 bats_update_forklift: "yes"
 bats_tests:
-  - "fb-install-katello.bats"
```

Comment:
If you look at the contents it's much more installation and much less testing. I'd prefer to write some proper tests and leave installation to ansible roles.